fem/pgridfunc.hpp

```diff
@@ -90,15 +90,23 @@ class ParGridFunction : public GridFunction
        @note Defining this method overwrites the implicitly defined copy
        assignment operator. */
    ParGridFunction &operator=(const ParGridFunction &rhs)
-   { return operator=((const Vector &)rhs); }
+   {
+      return operator=((const Vector &)rhs);
+   }
 
    /// Assign constant values to the ParGridFunction data.
    ParGridFunction &operator=(double value)
-   { GridFunction::operator=(value); return *this; }
+   {
+      GridFunction::operator=(value);
+      return *this;
+   }
```

Comment:
Why these style changes?

Reply:
My bad, it's probably a bad rebase from my side.